src/classes/PMT_PaymentWizard_TEST.cls

```diff
@@ -669,6 +669,64 @@ private class PMT_PaymentWizard_TEST {
         System.assert(redirect);
     }
 
+    /*********************************************************************************************************
+    * @description Verify if Warning message is displayed in Schedule Payment when Automatic Payment is disabled
+    * in Payment custom settings
+    */
+    @isTest
+    private static void shouldDisplayWarningInSchedulePaymentWhenAutoPaymentDisabled() {
+        npe01__Contacts_And_Orgs_Settings__c PaymentsSettingsForTests = UTIL_CustomSettingsFacade.getContactsSettingsForTests(
```

Comment:
There's actually no need to assign this to a var since the method isn't using the `PaymentsSettingsForTests` var anyway. Just call the method without the assignment. Same for the other new test methods.

Reply:
I didn't notice it, @force2b , will take out the assignment.